openshift/rh-che.config.yaml

```diff
@@ -72,3 +71,9 @@ data:
   CHE_WORKSPACE_STORAGE: "/home/user/che/workspaces"
   CHE_WORKSPACE_PLUGIN__BROKER_VSCODE_IMAGE: "eclipse/che-vscode-extension-broker:v0.13.0"
   CHE_FABRIC8_CDN_PREFETCH_EDITOR: 'NULL'
+  CHE_CORE_JSONRPC_PROCESSOR__MAX__POOL__SIZE: '60'
+  CHE_CORE_JSONRPC_PROCESSOR__CORE__POOL__SIZE: '10'
+  CHE_CORE_JSONRPC_PROCESSOR__QUEUE_CAPACITY: '30000'
+  CHE_CORE_JSONRPC__MINOR__PROCESSOR__MAX__POOL__SIZE: '60'
+  CHE_CORE_JSONRPC__MINOR__PROCESSOR__CORE__POOL__SIZE: '5'
+  CHE_CORE_JSONRPC__MINOR__PROCESSOR__QUEUE__CAPACITY: '10000'
```

Comment:
is this config optimal for 4GB RAM for che server assuming that  `-XX:MaxRAMFraction=2 ` is set for che server java opts

Reply:
The latest values have been agreed on by Sergii Kabashniuk.
They should be correct for the server from version 7-beta-2 and forwards.